src/com/sap/piper/cm/ChangeManagement.groovy

```diff
@@ -63,7 +64,7 @@ public class ChangeManagement implements Serializable {
     }
 
     boolean isChangeInDevelopment(String changeId, String endpoint, String credentialsId, String clientOpts = '') {
-        int rc = executeWithCredentials(BackendType.SOLMAN, endpoint, credentialsId, 'is-change-in-development', ['-cID', "'${changeId}'", '--return-code'],
+        int rc = executeWithCredentials(BackendType.SOLMAN, '', [], endpoint, credentialsId, 'is-change-in-development', ['-cID', "'${changeId}'", '--return-code'],
```

Comment:
Should be filled according to the cm client docker image. Needs to be implemented in executeWithCredentials, of course. Should be done IMO in another PR.